### PR TITLE
Document the deploy command

### PR DESCRIPTION
**Pull request by ironcladlou**

### CODE REVIEW

**Comment by ironcladlou** [comment]
cc @adellape @abhgupta

**Comment by ironcladlou** [comment]
I want to get the stage set for bringing the rollback and cancelation docs from the CLI guide into this doc in a separate PR.

File: dev_guide/deployments.adoc [hunk 1]
```diff
@@ -110,28 +110,61 @@ created each time a new version of the `origin-ruby-sample:latest` image reposit
 ====
 
 == Deployment Client Tool Operations
```
Comment (adellape) [inline dev_guide/deployments.adoc]: To better match other Dev Guide topics, how about either re-titling this "Using the CLI" (a la [1]), or maybe better yet just dropping the heading & intro sentence and let them be their own `==`-level headings (a la [2][3]).

[1] http://docs.openshift.org/latest/dev_guide/new_app.html
[2] http://docs.openshift.org/latest/dev_guide/projects.html
[3] http://docs.openshift.org/latest/dev_guide/builds.html

File: dev_guide/deployments.adoc [hunk 2]
```diff
@@ -110,28 +110,61 @@ created each time a new version of the `origin-ruby-sample:latest` image reposit
...
 
 ----
 $ oc deploy <deployment_config> [--options]
 ----
 
-[cols= 2]
-|===
-|`oc deploy <deployment_config> --latest`
-|Manually starts the specified deployment
+=== Start a Deployment [[start-deployment]]
```
Comment (adellape) [inline dev_guide/deployments.adoc]: Can we switch these to gerund verbs, "Starting a Deployment", "Viewing a Deployment", etc?

**Comment by adellape** [comment]
@ironcladlou Comments inline, otherwise LGTM + thanks for doin it!

File: dev_guide/deployments.adoc (hunk 1 above)
Comment (ironcladlou) [inline dev_guide/deployments.adoc]: Agree, much cleaner. Changed.

File: dev_guide/deployments.adoc (hunk 2 above)
Comment (ironcladlou) [inline dev_guide/deployments.adoc]: Sure. Done, check it out.

**Comment by ironcladlou** [comment]
Feedback addressed, PTAL.

**Comment by adellape** [comment]
:heart: it.

**Pull request merged**